indico/modules/events/management/templates/lock_event.html

```diff
@@ -1,28 +1,13 @@
-{% extends 'layout/base.html' %}
+{% from 'confirmation_dialog.html' import confirmation_dialog %}
 
-{% block title %}{% trans %}You are about to lock this event!{% endtrans %}{% endblock %}
+{% set confirmation_message %}
+    {% trans -%}
+        Please note that if you lock this event, you will not be able to change it anymore.
+        Only the event owner or an administrator will be able to unlock it.
```

Comment:
"Only the creator of the event or a category manager will be able to unlock it."